native_client/java/app/src/main/java/deepspeech/mozilla/org/deepspeech/Model.java

```diff
@@ -0,0 +1,43 @@
+package deepspeech.mozilla.org.deepspeech;
+
+public class Model {
+
+    SWIGTYPE_p_p_ModelState _mspp;
```

Comment:
`SWIGTYPE_*` is not very nice to expose, but is this really such a big deal ?

Reply:
I see it as a bug in your SWIG binding definition, but can be left as a FIXME for now.